kani-compiler/src/kani_middle/attributes.rs

```diff
@@ -138,11 +149,68 @@ pub fn extract_harness_attributes(tcx: TyCtxt, def_id: DefId) -> HarnessAttribut
                 // Internal attribute which shouldn't exist here.
                 unreachable!()
             }
+            KaniAttributeKind::Ensures | KaniAttributeKind::Requires => {
+                todo!("Contract attributes are not supported on proofs (yet)")
```

Comment:
I don't understand what that means for the user. We may need a better message.

Reply:
+1... Please add a user friendly message with `sess.error()`. Also, do we even expect contracts to be added to proof harnesses? This message is setting an expectation that hasn't been discussed.